exporter/azure_monitor/common/common.go

```diff
@@ -12,7 +12,7 @@ var AzureMonitorContext = map[string]interface{} {
 	"ai.cloud.roleInstance": getHostName(),
 	"ai.device.id": getHostName(),
 	"ai.device.type": "Other",
-	"ai.internal.sdkVersion":  "go:oc0.1",
+	"ai.internal.sdkVersion":  "go0.1:oc0.1:exp0.1",

```

Comment:
Curious, what does go0.1 mean, and what does oc0.1 mean?

Reply:
I was following the python repo example py3.7.3:oc0.6.0:ext0.2.0, but I must have not been thinking actually. It makes no sense to say "go0.1" since I am not updating the go language or opencensus. I will change to "go:exp0.1" where exp is my exporter version.

Reply:
This makes more sense, thanks.